pandas/core/reshape/melt.py

```diff
@@ -420,7 +420,8 @@ def melt_stub(df, stub, i, j, value_vars, sep):
         newdf = melt(df, id_vars=i, value_vars=value_vars,
                      value_name=stub.rstrip(sep), var_name=j)
         newdf[j] = Categorical(newdf[j])
-        newdf[j] = newdf[j].str.replace(re.escape(stub + sep), "")
+        newdf[j] = newdf[j].str.replace(re.escape(stub + sep), "",
```

Comment:
Thanks very much for taking your time to reivew @jreback . Yes, there is test in tests/reshape/test_melt.py that will fail, because previously the default is True, and now is None. And apparently, in this melt, we want a True for regex.